.github/workflows/CI.yml

```diff
@@ -48,10 +48,13 @@ jobs:
           key: hardhatNodeModules-${{ runner.os }}-${{ hashFiles('**/hardhat/yarn.lock') }}
 
       - run: yarn --frozen-lockfile
+      - run: yarn cypress install
```

Comment:
I feel like we should create a separate GitHub action for running e2e tests vs adding more stuff to this one. That way cypress tests failing will still let people see typecheck/ linting pass separately. Also will help cut down the time CI action is running, esp as we start adding more cypress tests

Reply:
we could split out linting, but i think it's good to run all the tests together. we'll need it for a combined coverage metric

Reply:
Yea thats what I meant, linting + typecheck in one, E2E and unit testing in another